Sources/protoc-gen-swift/MessageGenerator.swift

```diff
@@ -327,6 +327,10 @@ class MessageGenerator {
     // Nested messages
     //
 
+    for o in oneofs {
+      o.generateRuntimeSupport(printer: &p)
```

Comment:
_Extensions_ is always an overloaded terms in protos and Swift…  If we have a better name let's bulk do it after all these PRs land.